### DISCUSSION TITLE
Install hangs at Applying thehive Salt State

### DISCUSSION THREAD

**Discussion by Sazerak** [Security-Onion-Solutions/securityonion#2377] [Unsupported Versions] [answered]
I noticed that this seems to be a somewhat common issue, but I haven't noticed a fix for it other than not installing the Hive.

I'm installing this from the ISO on ESXi 6.7 (HP DL380 Gen7). Gave it 4 CPUs, 16Gigs of RAM and 200GB HDD.

Here is the tail end of the log file:

[INFO    ] Completed state [/opt/so/conf/so-status/so-status.conf] at time 15:55:29.652969 (duration_in_ms=8.65)
[INFO    ] Running state [cortexscript] at time 15:55:29.656590
[INFO    ] Executing state cmd.script for [cortexscript]
[INFO    ] Fetching file from saltenv 'base', ** done ** 'thehive/scripts/cortex_init'
[INFO    ] Executing command '/opt/so/__salt.tmp.vqb99i8y' in directory '/opt/so'

Any help is appreciated.

**Comment by TOoSmOotH** [comment, marked as answer]
Can you grep ERROR in the sosetup.log and paste that?

**Reply by Sazerak** [reply]
Thanks for your reply. I actually changed gears for now as I'm not even able to install it properly without theHive (plus, I forgot to get the log file for this attempt before reverting my snapshot). I started a new discussion [HERE](https://github.com/Security-Onion-Solutions/securityonion/discussions/2382) with my most recent issue.

**Reply by Sazerak** [reply]
Just wanted to update that the issue is no longer present. Maybe the changes I did in relation to my other problem fixed this as well, but either way, all seems good now.

**Reply by Sazerak** [reply]
@FYusu did you read my comments on the other post I made (linked above)? The tl;dr is that my default route was the monitoring interface, and that is what was causing all the issues I was having. By changing the network configurations prior to running the SO configuration wizard, all issues were solved, including the issue of being stuck at Applying The Hive. Hope this helps!
